Vagrantfile

```diff
@@ -227,6 +227,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       v.memory            = vagrant_openshift_config['memory'].to_i
       v.cpus              = vagrant_openshift_config['cpus'].to_i
       v.customize ["modifyvm", :id, "--cpus", vagrant_openshift_config['cpus'].to_s]
+      full_provision(override.vm)
```

Comment:
completely agree. the problem was this image is really outdated and has some issues with being in sync with the docs. PATH is not exported and so on, so basically people kept modifying the docs with ./oadm or ./openshift.
there was a talk yesterday and today about switching to a fedora 23 or centos 7 image for vagrant, what do you think?